### PR TITLE
v3.0/how-to/scale/with-ansible:add_attention_pd

### PR DESCRIPTION
**Pull request by superlzs0476**
### What is changed, added or deleted? 

Add a note to PD's scale-up step

### What is the related PR or file link(s)? 

N/A

### Which version does your change affect? 

Just version of v3.0

### CODE REVIEW

**Comment by superlzs0476** [comment]
@dcalvin @rleungx  PTAL

File: v3.0/how-to/scale/with-ansible.md [hunk 1]
```diff
@@ -213,6 +213,10 @@ TiDB 集群可以在不影响线上服务的情况下进行扩容和缩容。
         {deploy_dir}/scripts/start_pd.sh
         ```
 
+    > **注意：**
+    >
+    > 启动前，需要通过 [PD Control](https://pingcap.com/docs-cn/stable/reference/tools/pd-control/) 工具确认当前 PD `health` 状态均为 “true"，否则启动新 PD 会失败，同时日志中会报错 `["join meet error"] [error="etcdserver: unhealthy cluster"]`。
```
Comment (dcalvin) [inline v3.0/how-to/scale/with-ansible.md]: > **注意：**
        >
        > 启动前，需要通过 [PD Control](https://pingcap.com/docs-cn/stable/reference/tools/pd-control/) 工具确认当前 PD `health` 状态均为 “true"，否则启动新 PD 会失败，同时日志中会报错 `["join meet error"] [error="etcdserver: unhealthy cluster"]`。

File: v3.0/how-to/scale/with-ansible.md [hunk 2]
```diff
@@ -213,6 +213,10 @@ TiDB 集群可以在不影响线上服务的情况下进行扩容和缩容。
         {deploy_dir}/scripts/start_pd.sh
         ```
 
+        > **注意：**
+        >
+        > 启动前，需要通过 [PD Control](https://pingcap.com/docs-cn/stable/reference/tools/pd-control/) 工具确认当前 PD `health` 状态均为 “true"，否则启动新 PD 会失败，同时日志中会报错 `["join meet error"] [error="etcdserver: unhealthy cluster"]`。
```
Comment (lilin90) [inline v3.0/how-to/scale/with-ansible.md]:
```suggestion
        > 启动前，需要通过 [PD Control](https://pingcap.com/docs-cn/stable/reference/tools/pd-control/) 工具确认当前 PD `health` 状态均为 "true"，否则启动新 PD 会失败，同时日志中会报错 `["join meet error"] [error="etcdserver: unhealthy cluster"]`。
```

File: v3.0/how-to/scale/with-ansible.md [hunk 3]
```diff
@@ -213,6 +213,10 @@ TiDB 集群可以在不影响线上服务的情况下进行扩容和缩容。
         {deploy_dir}/scripts/start_pd.sh
         ```
 
+        > **注意：**
+        >
+        > 启动前，需要通过 [PD Control](https://pingcap.com/docs-cn/stable/reference/tools/pd-control/) 工具确认当前 PD `health` 状态均为 "true"，否则启动新 PD 会失败，同时日志中会报错 `["join meet error"] [error="etcdserver: unhealthy cluster"]`。
```
Comment (lilin90) [inline v3.0/how-to/scale/with-ansible.md]:
```suggestion
        > 启动前，需要通过 [PD Control](/v3.0/reference/tools/pd-control.md) 工具确认当前 PD `health` 状态均为 "true"，否则启动新 PD 会失败，同时日志中会报错 `["join meet error"] [error="etcdserver: unhealthy cluster"]`。
```

**Pull request merged**